main.py

```diff
@@ -1,7 +1,49 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    for genre in ["Western", "Action", "Dramma"]:
+        Genre.objects.create(
+            name=genre
+        )
+
+    for actor in ["George Klooney", "Kianu Reaves",
+                  "Scarlett Keegan", "Will Smith",
+                  "Jaden Smith", "Scarlett Johansson"]:
```

Comment:
Consider using a list of tuples for actors, where each tuple contains the first and last name. This would eliminate the need to split the names later.